backend/contracts/Voting.sol

```diff
@@ -108,6 +111,10 @@ contract Voting is Ownable {
         voters[msg.sender].hasVoted = true;
         proposalsArray[_id].voteCount++;
 
+         if (proposalsArray[_id].voteCount > proposalsArray[_id].voteCount) {
+            winningProposalID = _id;
+        }
```

Comment:
to fix

Reply:
remove tallyVotes function too

Reply:
Then test the contract in remix IDE

Reply:
Done in remix, but forgot remove tallyVotes